R/print_table.R

```diff
@@ -0,0 +1,13 @@
+print_table <- function(data, columns) {
+  # Makes a pretty table with the given data and labels.
+  #
+  # Args:
+  #   the data to be displayed and the column names
+  # Returns:
+  #   a kable
+  fig <- round(prop.table(table(data))*100, digits = 1)
+  fig <- as.data.frame(fig)
+  names(fig) <- columns
+
+  return(knitr::kable(fig, booktabs = T) %>% kableExtra::kable_styling(latex_options = "striped"))
```

Comment:
do we need to add knitr or kableExtra to the imports?

Reply:
Yes, we do. I'll add them after your review.